### PR TITLE
Temporary fix for broken spatial selection

### PR DESCRIPTION
**Pull request by rod-glover**
Addresses temporarily #198 by falling back to an ncWMS-PCIC instance for `GetFeatureInfo` requests.

### CODE REVIEW

File: pdp/static/js/pdp_raster_map.js [hunk 1]
```diff
@@ -126,18 +126,49 @@ function intersection(b1, b2) {
...
-    real_bounds.extend(new OpenLayers.LonLat(parseFloat(bbox.attributes.getNamedItem('maxx').value), parseFloat(bbox.attributes.getNamedItem('maxy').value)));
+    const real_bounds = new OpenLayers.Bounds();
+    const bboxes = capabilities.find(
+      'Capability > Layer > Layer > Layer > LatLonBoundingBox'
+    );
+    // Since the dataset <Name> is now the filepath, which does not necessarily
+    // match `layer_name`, we can't find it that way. Instead we make the
+    // somewhat fragile but currently valid assumption that the first (and
+    // only) such layer is the one we want.
+    const bbox = bboxes[0];
```
Comment (rod-glover) [inline pdp/static/js/pdp_raster_map.js:157]: remove

File: pdp/static/js/pdp_raster_map.js [hunk 2]
```diff
@@ -126,18 +126,49 @@ function intersection(b1, b2) {
...
+    console.log('### getRasterBbox', {bboxes})
```
Comment (rod-glover) [inline pdp/static/js/pdp_raster_map.js:158]: remove

File: pdp/static/js/pdp_raster_map.js [hunk 3]
```diff
@@ -126,18 +126,49 @@ function intersection(b1, b2) {
...
+        parseFloat(bbox.attributes.getNamedItem('miny').value)
+      )
+    );
+    real_bounds.extend(
+      new OpenLayers.LonLat(
+        parseFloat(bbox.attributes.getNamedItem('maxx').value),
+        parseFloat(bbox.attributes.getNamedItem('maxy').value)
+      )
+    );
+    console.log('### getRasterBbox', {real_bounds})
```
Comment (rod-glover) [inline pdp/static/js/pdp_raster_map.js:171]: remove

File: pdp/static/js/pdp_raster_map.js [hunk 4]
```diff
@@ -126,18 +126,49 @@ function intersection(b1, b2) {
...
-        srs = bbox.attributes.getNamedItem('SRS');
-    return new OpenLayers.Projection(srs.value);
+    const bboxes = capabilities.find(
+      'Capability > Layer > Layer > Layer > BoundingBox'
+    );
+    // Since the dataset <Name> is now the filepath, which does not necessarily
+    // match `layer_name`, we can't find it that way. Instead we make the
+    // somewhat fragile but currently valid assumption that the first (and
+    // only) such layer is the one we want.
+    const bbox = bboxes[0];
```
Comment (corviday) [inline pdp/static/js/pdp_raster_map.js:136]: Just to make sure I understand what's going on here, the thing we're assuming is that all layers / variables in a file have the same spatial extent, right?

File: pdp/static/js/pdp_raster_map.js [hunk 5]
```diff
@@ -126,18 +126,49 @@ function intersection(b1, b2) {
...
+    console.log('### getRasterNativeProj', {bboxes})
+    const srs = bbox.attributes.getNamedItem('SRS');
+    // For reasons beyond mortal ken, ncWMS now returns CRS:84 instead of
+    // EPSG:4326 for many of our datasets. CRS:84 is not recognized by proj4js.
+    const value = srs.value === "CRS:84" ? "EPSG:4326" : srs.value;
+    console.log('### getRasterNativeProj', {value})
```
Comment (corviday) [inline pdp/static/js/pdp_raster_map.js:142]: If you're removing console.log statements, there's one here too.

**Review by corviday — APPROVED** [review]
This seems like a good stopgap measure to me.

File: pdp/static/js/pdp_raster_map.js (hunk 4 above)
Comment (rod-glover) [reply pdp/static/js/pdp_raster_map.js:136]: Yes. That is one way of putting it, and possibly a better one.

**Comment by rod-glover** [comment]
Verified working at https://services.pacificclimate.org/dev/portal/bc_prism/map/ etc.

**Pull request merged**